source/core/parameters_lagrangian.cc

```diff
@@ -1240,7 +1242,9 @@ namespace Parameters
           point_on_rotation_axis.insert({counter, Point<3>(zero_tensor)});
         }
 
+      // NOTE This first vector should not be initialized this big.
       outlet_boundaries.reserve(DEM_BC_number);
+      bc_types.reserve(DEM_BC_number);
```

Comment:
Yeah but how big is DEM_BC_number? Here it is reserved and not initialized btw, so this is not really a big deal...

Reply:
You are right, DEM_BC_number is really small